docsrc/imap/concepts/deployment/databases.rst

```diff
@@ -61,7 +61,7 @@ The format of each record is as follows::
 
     Data: <Type Number>SP<Partition>SP<ACL (space-separated userid/rights pairs)>
 
-File type can be: `twoskip`_ (default), `flat`_, `skiplist`_, or `sql`_.
+File type can be: `twoskip`_ (default), `twom`_, `flat`_, `skiplist`_, or `sql`_.
```

Comment:
Before the current change in this file after the default type, the remaining formats were sorted in alphabetical order.

Reply:
Yes, good catch - I have fixed this for all types